upup/pkg/fi/cloudup/do/cloud.go.new

```diff
@@ -0,0 +1,566 @@
+/*
```

Comment:
This file should not be added then removed. Please edit it out of the commit stream.

Reply:
Sure, Removed it.